crates/ruff_python_ast/ast.toml

```diff
@@ -80,39 +80,367 @@ add_suffix_to_is_methods = true
 anynode_is_label = "expression"
 rustdoc = "/// See also [expr](https://docs.python.org/3/library/ast.html#ast.expr)"
 
-[Expr.nodes]
-ExprBoolOp = {}
-ExprNamed = {}
-ExprBinOp = {}
-ExprUnaryOp = {}
-ExprLambda = {}
-ExprIf = {}
-ExprDict = {}
-ExprSet = {}
-ExprListComp = {}
-ExprSetComp = {}
-ExprDictComp = {}
-ExprGenerator = {}
-ExprAwait = {}
-ExprYield = {}
-ExprYieldFrom = {}
-ExprCompare = {}
-ExprCall = {}
-ExprFString = {}
-ExprStringLiteral = {}
-ExprBytesLiteral = {}
-ExprNumberLiteral = {}
-ExprBooleanLiteral = {}
-ExprNoneLiteral = {}
-ExprEllipsisLiteral = {}
-ExprAttribute = {}
-ExprSubscript = {}
-ExprStarred = {}
-ExprName = {}
-ExprList = {}
-ExprTuple = {}
-ExprSlice = {}
-ExprIpyEscapeCommand = {}
+[Expr.nodes.ExprBoolOp]
+rustdoc = "/// See also [BoolOp](https://docs.python.org/3/library/ast.html#ast.BoolOp)"
+
+[[Expr.nodes.ExprBoolOp.fields]]
+name = "op"
+type = "BoolOp"
+
+[[Expr.nodes.ExprBoolOp.fields]]
+name = "values"
+type = "Expr"
+seq = true
+
+[Expr.nodes.ExprNamed]
+rustdoc = "/// See also [NamedExpr](https://docs.python.org/3/library/ast.html#ast.NamedExpr)"
```

Comment:
Nit: Just doc?

Reply:
`rustdoc` is consistent with the existing per-group comments.  If we use `doc` here we should change the other one to be consistent.